cpp/test/Glacier2/sessionHelper/Client.cpp

```diff
@@ -235,11 +235,11 @@ Client::run(int argc, char** argv)
     _initData.properties = Ice::createProperties(argc, argv, communicator->getProperties());
     _initData.properties->setProperty("Ice.Default.Router", "Glacier2/router:" + getTestEndpoint(50));
 
-    Dispatcher dispatcher;
-    auto dfut = std::async(launch::async, [&dispatcher] { dispatcher.run(); });
+    Executor executor;
+    auto dfut = std::async(launch::async, [&executor] { executor.run(); });
```

Comment:
We should rename `dfut` variable to something more meaningful. `executorFuture`?

Reply:
Fixed